helpers/adapters_tracker.go

```diff
@@ -3,19 +3,23 @@ package helpers
 import (
 	"context"
 	"sync"
+
+	"github.com/layer5io/meshery/models"
 )
 
 // AdaptersTracker is used to hold the list of known adapters
 type AdaptersTracker struct {
-	adapters     map[string]struct{}
+	adapters     map[string]models.Adapter
```

Comment:
Do we have a struct that represents a Meshery deployment?

Reply:
U mean an incluster meshery instance? No I dont think we do. @leecalcote

Reply:
Ok. I see the preferences struct the you pointed out. This is an area for rework when Environments come...